messaging/src/main/java/io/lantern/messaging/metadata/Metadata.kt

```diff
@@ -153,9 +153,12 @@ class Metadata(val mimeType: String?, val thumbnail: ByteArray?, val thumbnailMi
         }
 
         private fun audioMetadata(file: File, defaultMimeType: String): Metadata {
+            val duration = MediaMetadataRetriever().apply {
```

Comment:
Later in this same function we already decode the file, so extracting metadata here is inefficient.